host/lib/transport/libusb1_zero_copy.cpp

```diff
@@ -139,7 +139,7 @@ class libusb_zero_copy_mb : public managed_buffer
     {
         if (wait_for_completion(timeout))
         {
-            if (result.status != LIBUSB_TRANSFER_COMPLETED)
+            if (result.status != LIBUSB_TRANSFER_COMPLETED || result.status != LIBUSB_TRANSFER_CANCELLED)
```

Comment:
This should be && right?

Reply:
Yeah i think it should be && but when i run this code in company it's working in fact it was my teammate code. let me check it. I will tell you when i checked it at saturday. I'm not sure right now maybe something else is wrong.

Reply:
> I tried running this code and it always throws an exception. Look at the clause again:
> 
> ```c++
> if (result.status != LIBUSB_TRANSFER_COMPLETED || result.status != LIBUSB_TRANSFER_CANCELLED) {}
> ```
> 
> Let's say result.status returns normally, then `result.status == LIBUSB_TRANSFER_COMPLETED`. In that case, `result.status != LIBUSB_TRANSFER_CANCELLED` is true, and we throw an exception. So it needs to be &&.

Yes i got it.It was distracting.